GOVERNANCE.md

```diff
@@ -0,0 +1,192 @@
+
+In repositories using an OWNERS file, Maintainers are listed as Approvers in that file.
+
+A Maintainer must meet the responsibilities and requirements of a Reviewer, plus:
+* Responsibilities include:
+    * Sustained high level of reviews of pull requests to the project or subproject, with a goal of one or more a week when averaged across the year.
+    * Merging pull requests which pass review
+    * Mentoring new Reviewers
+    * Participating in CNCF maintainer activities for the projects they are maintainers of
+    * Assisting Core Maintainers in determining strategy and policy for the project
+    * Participating in, and leading, community meetings
+* Requirements
+    * Experience as a Reviewer for at least 6 months, or status as an Emeritus Maintainer. The time requirement may be overridden by a supermajority vote of Maintainers and Core Maintainers.
+    * Demonstrates a broad knowledge of the project or one or more of its subprojects
+    * Is able to exercise judgment for the good of the project, independent of their employer, friends, or team
+    * Mentors contributors, reviewers, and new maintainers
+    * Collaborates with other Maintainers to work on complex contributions
+    * Can commit to maintaining a high level of contribution to the project or one of its subprojects
+* Additional privileges:
+    * Represent the project in public as a senior project member
+	* Represent the project in interactions with the CNCF
+    * Have a voice, but not a vote, in Core Maintainer decision-making meetings
+
+#### Process of becoming a maintainer:
+1. A current reviewer must be sponsored by a Maintainer of the repository in question or a Core Maintainer. The Maintainer or Core Maintainer will open a PR against the repository and add the nominee as a Maintainer in the MAINTAINERS.md file. The need for a sponsor is removed for Emeritus Maintainers, who may open this pull request themselves.
+2. The nominee will add a comment to the PR confirming that they agree to all requirements and responsibilities of becoming a Maintainer.
+3. A majority of the current Maintainers of the repository or subproject (including Core Maintainers) must then approve the PR. The need for a majority is removed for Emeritus Maintainers, who require only 2 current Maintainers or Core Maintainers to approve their return.
+
+### Core Maintainer
+Description: As the Podman project is composed of a number of subprojects, most maintainers will not have full knowledge of the full project and all its technical aspects. Those that do are eligible to become Core Maintainers, responsible for decisions affecting the entire project. Core Maintainers may act as a maintainer in all repositories and subprojects of the Podman Project.
+
+* Responsibilities include:
+    * All responsibilities of a maintainer
+    * Determining strategy and policy for the project
+    * Participating in, and leading, community meetings
```

Comment:
```suggestion
    * Participating in, and leading, community meetings
    * Communicate with the CNCF on behalf of the project
```

Reply:
Already says "All responsibilities of a Maintainer" which includes that. Though I should remove the "Participating in community meetings bit" as it's duplicated from the Maintainer list